wasm/jit/engine.go

```diff
@@ -17,9 +17,9 @@ type engine struct {
 	// The actual Go-allocated stack.
 	// Note that we NEVER edit len or cap in JITed code so we won't get screwed when GC comes in.
 	stack []uint64
-	// Wasm stack pointer on .stack field which is accessed by currentStackBasePointer+currentStackPointer
-	currentStackPointer     uint64
-	currentStackBasePointer uint64
+	// Wasm stack pointer on .stack field which is accessed by stackBasePointer+stackPointer
+	stackPointer     uint64
+	stackBasePointer uint64
```

Comment:
add a comment on base pointer? since it is referenced in the above...

Reply:
ex what is the base? is it the program running's initial address? (just sayin)

Reply:
https://github.com/tetratelabs/wazero/pull/88/commits/066b4b5f0ae8a53340327ce0f4871689147b94b7 commented (note that "base pointer" or stack base pointer is a general term in calling conventions)